docs/governance/governance_model_v1_archive.rst

```diff
@@ -0,0 +1,263 @@
+* Technical decisions - for example, frameworks to adopt or key strategic objectives. Where there's no clear consensus from the community or the suggestions being made could be detrimental to the long term vision for the project, the Project Lead can determine the path to be taken
+* Feature prioritisation - if a particular feature needs to be prioritised, the Project Lead can instruct Acquia employees to work on developing that feature  
+
+Finance and budget
+******************
+
+Mautic now uses the :xref:`Mautic open collective` to manage its finances.
+
+Each year, Team Leads make a budget request for the coming year, which are reviewed by the Community Council. A Community Budget is prepared and adopted by the council.
+
+Transactions on the Open Collective are open and visible to all.
+
+About Mautic Core Team
+**********************
+
+Development is open and available to any member of the Mautic community. All fixes and improvements are done through pull requests to the code. This code is open source and publicly available. Pull requests and code submissions are decided upon by the Release Leader and the Core Team. When a decision isn't clearly evident then the following voting process will be implemented.
+
+Who are the Mautic Core Maintainers and what do they do?
+********************************************************
+
+The Mautic Core Team - who form part of the Product Team - is divided into 5 groups. Each team member can belong to only one group at a time. Any privilege listed for a particular group is also available to all higher priority groups. The Mautic Core groups - in descending order of priority - are as follows:
+
+The Project Lead
+================
+
+The Project Lead elects members into any other group, oversees project vision and direction, and makes decisions on proposed changes. The Project Lead listens to the counsel of trusted advisors and individuals respected for their contributions to Mautic. The Project Lead is appointed by Acquia.
+
+The full responsibilities and expectations of the Project Lead are detailed :ref:`on this page<community leadership project lead definition>`.
+
+The Technical Community Lead
+============================
+
+The Technical Community Lead supports the technical growth and development of the Mautic project and community. This role is pivotal in empowering other developers in Mautic Community, and setting up the foundations for them to become active contributors.
+
+They're the face of the developer community, and a central point of call for any technical issues.
+
+The Technical Community Lead also has primary responsibility for supporting in-person sprints, releases, and being a bridge between the community and Acquia from the technical perspective.
+
+.. There's no link to "/community-leadership/leadership-role-definitions#community-manager".
+
+The full responsibilities and expectations of the Technical Community Lead are detailed [here].
```

Comment:
There's no link to `/community-leadership/leadership-role-definitions#community-manager`.
Where should this linked to?

Reply:
I've added the link.